pkg/driver/driver.go

```diff
@@ -98,14 +100,19 @@ func Boot(ctx context.Context, d Driver, pw progress.Writer) (*client.Client, st
 		}
 		try++
 		if info.Status != Running {
-			if try > 2 {
+			if try > 3 {
 				return nil, "", errors.Errorf("failed to bootstrap %T driver in attempts", d)
 			}
 			if err := d.Bootstrap(ctx, func(s *client.SolveStatus) {
 				if pw != nil {
 					pw.Status() <- s
 				}
-			}); err != nil {
+			}); err != nil && (strings.Contains(err.Error(), "already exists") || strings.Contains(err.Error(), "not found")) {
+				// This most likely means another build is running in parallel
+				// Give it just enough time to finish creating resources then retry
+				time.Sleep(250 * time.Millisecond)
```

Comment:
will this always complete in <1s (i.e. 3 attempts w/ 250ms sleeps per attempt)?

Reply:
Not necessarily.  The new retry wont add more than 1s, but bootstrapping the builder when things are working properly can take far longer if you have to pull images, etc.